src/main/java/teammates/ui/template/InstructorFeedbackSessionActions.java

```diff
@@ -63,10 +60,6 @@ public InstructorFeedbackSessionActions(PageData data, FeedbackSessionAttributes
         
         this.allowedToSubmit = (session.isVisible() || session.isPrivateSession()) && shouldEnableSubmitLink;
         this.allowedToRemind = session.isOpened() && instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
-
-        this.toggleDeleteFeedbackSessionParams = "'" + Sanitizer.sanitizeForJs(courseId) + "','"
-                                               + Sanitizer.sanitizeForJs(feedbackSessionName) + "'";
-        this.toggleRemindStudentsParams = "'" + Sanitizer.sanitizeForJs(feedbackSessionName) + "'";
```

Comment:
Make sure that the two params `courseId` and `feedbackSessionName` are sanitized for js prior to this.